.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest an idea for the ProGit2 book or repository
+title: ""
+labels: ""
```

Comment:
```suggestion
labels: "enhancement"
```

Optional: automatically attach the `enhancement` label to feature requests.

We could also create and attach a `feature` label instead.

Reply:
I think `enhancement` makes the most sense here. We don't exactly have "features."